packages/dso-toolkit/src/components/card/card.args.ts

```diff
@@ -151,6 +129,6 @@ export function cardArgsMapper<TemplateFnReturnType>(
         }
       : undefined,
     content,
-    dsoCardClicked: (e) => a.dsoCardClicked?.(e.detail),
+    dsoCardClick: (e) => a.dsoCardClick(e.detail),
```

Comment:
Nu ik dit zie, vraag ik mij af of we elkaar eerder wel begrepen hebben 🤔 

Ik heb ergens een commentaar geplaatst over een Storybook Args event listener, dat deze niet optioneel hoeft te zijn omdat Storybook deze altijd voor ons maakt.

Heb jij mogelijk het Model aangepast en daar de optionaliteit weggehaald? Want om het Model zijn event listeners altijd optioneel.

Reply:
Idd. 

Evenwel blijft de verwarring bij mij toenemen. Ik doe een poging het helder te krijgen.

1. een Storybook Args event listener hoeft/is niet optioneel
2. een Model event listener is altijd optioneel

Alhier gaat het om een Storybook Args event listener, immers `card.args.ts`, dus niet optioneel.

Reply:
Klopt!

🚌

Reply:
Met Eric gesproken.